{{cookiecutter.project_name}}/docker/caddy/ci.sh

```diff
@@ -6,15 +6,15 @@ set -o nounset
 
 run_ci () {
   # Validating:
-  caddy validate
+  caddy validate --config /etc/caddy/Caddyfile
```

Comment:
Let's define `CADDYFILE='/etc/caddy/Caddyfile'` at the top of the file and then use it: 

```suggestion
  caddy validate --config "$CADDYFILE"
```

Reply:
And in other places

Reply:
should be good now, commited